source/helpers/open-tabs.ts

```diff
@@ -1,11 +1,19 @@
+import showToast from '../github-helpers/toast.js';
+import pluralize from '../helpers/pluralize.js';
+
 export default async function openTabs(urls: string[]): Promise<boolean> {
 	if (urls.length >= 10 && !confirm(`This will open ${urls.length} new tabs. Continue?`)) {
 		return false;
 	}
 
-	await browser.runtime.sendMessage({
+	const response = browser.runtime.sendMessage({
 		openUrls: urls,
 	});
 
+	await showToast(response, {
+		message: 'Opening…',
+		doneMessage: pluralize(urls.length, '$$ tab') + ' opened',
+	});
```

Comment:
The toast is now shown only if the user confirms the opening.

Also it acts as a notification on mobile, where opening tabs isn't as visible

https://github.com/refined-github/refined-github/assets/1402241/7406c973-6d51-43ad-baf1-08df23611d2f